validator/sawtooth_validator/server/network_handlers.py

```diff
@@ -242,6 +243,13 @@ def add(
         ),
         thread_pool)
 
+    # GOSSIP_MESSAGE ) Determines if this is a consensus message and notifies
+    # the consensus engine if it is
+    dispatcher.add_handler(
+        validator_pb2.Message.GOSSIP_MESSAGE,
+        GossipConsensusHandler(gossip=gossip, notifier=consensus_notifier),
+        thread_pool)
+
```

Comment:
Is it possible to assign high priority of consensus message handling. This might be required since consensus messages (eg: in case of Raft) needs to be processed at higher priority than rest of Gossip messages (block, batch) to make progress in the validator and to publish blocks sooner. For eg: In case of Raft, till all the followers gets the log entry, leader cannot commit the block. Similarly consensus messages like heart beat, election related messages has to be processed sooner compared to batch/block gossip messages. May be it might make sense to introduce a new message type like GOSSIP_CONSENSUS_MESSAGE and to set its priority high.

Reply:
This is a good point.